source/stylesheets/header.scss

```diff
@@ -30,14 +34,14 @@
 
   .topics-nav[open] & {
   	background-color: $dark-purple;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
```

Comment:
Looks like indentation is off in this block. You can also replace these two lines with bourbon's `@include border-bottom-radius(0)` now, if you'd like.